layouts/css/vendor/_prism-tomorrow.styl

```diff
@@ -0,0 +1,99 @@
+/**
+ * prism.js tomorrow night eighties for JavaScript, CoffeeScript, CSS and HTML
+ * Based on https://github.com/chriskempson/tomorrow-theme
+ * @author Rose Pritchard
+ */
+
+code[class*='language-'],
+pre[class*='language-']
+  color #ccc
```

Comment:
$light-gray2

Reply:
It's a 3rd-party file, I'd rather keep it close to the original.